include/repeated_hafnian.hpp

```diff
@@ -315,22 +408,24 @@ double hafnian_rpt_quad(std::vector<double> &mat, std::vector<int> &rpt) {
  *      `mat = [1]` and `rpt = [6]` represents a \f$6\times 6\f$ matrix of all ones.
  * @return loop hafnian of the input matrix
  */
-std::complex<double> loop_hafnian_rpt_quad(std::vector<std::complex<double>> &mat, std::vector<std::complex<double>> &mu, std::vector<int> &rpt) {
-    std::vector<std::complex<long double>> matq(mat.begin(), mat.end());
-    std::vector<std::complex<long double>> muq(mu.begin(), mu.end());
-    std::complex<long double> haf;
-    int s = std::accumulate(rpt.begin(), rpt.end(), 0);
-    int n = std::sqrt(static_cast<double>(mat.size()));
-
-    if ( s == 0 || n == 0)
-        haf = std::complex<long double>(1.0, 0.0);
-    else
-        haf = loop_hafnian_rpt(matq, muq, rpt);
-
-    return static_cast<std::complex<double>>(haf);
+std::complex<double>
```

Comment:
How come the return type is on a separate line here, but on the same line for the above functions?

Reply:
forgot to clang format this function :(